.husky/pre-commit

```diff
@@ -0,0 +1,2 @@
+prettier $(git diff --cached --name-only --diff-filter=ACMR | sed 's| |\\ |g') --write --ignore-unknown
```

Comment:
@Shryansh107 any reason you are not using ESLint  ? 
also can you add GitHub action to test linting in PRs and commits